cdk/README.md

```diff
@@ -426,13 +508,24 @@ const microservices = new Microservices(app, 'ConsulMicroservices', environment.
         environment: envProps.ecsEnvironment,
         serviceDescription: greeterDescription,
       });
+
+      // CONSUL CONNECT
+      greeter.connectTo(name, );
```

Comment:
There is nothing after comma. We can either write `greeter.connectTo(name, { local_bond_port: 3001 })` or `greeter.connectTo(name);`

Reply:
I incorporated this feedback in [here](https://github.com/aws-samples/amazon-ecs-fargate-consul-connect-example/pull/16/commits/61a8f4baffa4f4530652a283cb2666a163f2ae6b)